helm/hpcc/templates/_helpers.tpl

```diff
@@ -2245,12 +2245,10 @@ spec:
    {{- if hasKey .hpa "behavior" }}
   behavior: 
     {{- if hasKey .hpa.behavior "scaleDown" }}
-    scaleDown:
-     {{- include "hpcc.addHPABehaviorScale" (dict "behaviorScale" .hpa.behavior.scaleDown) | indent 6 -}}
+    scaleDown: {{- include "hpcc.addHPABehaviorScale" (dict "behaviorScale" .hpa.behavior.scaleDown) | indent 6 }}
```

Comment:
This looks like it should be using nindent rather than indent.  Same for line 2251.  Is it correct?

Reply:
Yes it works as-is, I tested several times before PR created, several other times after your comment. See output below. Spent more time shifting the whitechar controls for generated output until the nindent based output resulted in the same format as my original layout.
```---
# Source: hpcc/templates/esp.yaml
apiVersion: autoscaling/v2
kind: HorizontalPodAutoscaler
metadata:
  name: eclwatch-hpa
spec:
  scaleTargetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: eclwatch
  minReplicas: 1
  maxReplicas: 3
  behavior:
    scaleDown:
      stabilizationWindowSeconds: 300
      policies:
      - type: Pods
        value: 1
        periodSeconds: 20
      - type: Percent
        value: 10
        periodSeconds: 60
    scaleUp:
      stabilizationWindowSeconds: 0
      policies:
      - type: Percent
        value: 10
        periodSeconds: 60
      - type: Pods
        value: 3
        periodSeconds: 60
      selectPolicy: Min
  metrics:
  - type: Resource
    resource:
      name: cpu
      target:
        type: Utilization
        averageUtilization: 50
---